src/vdaf/hits.rs

```diff
@@ -437,92 +437,87 @@ impl<I: Idpf<2, 2>> Aggregator for Hits<I> {
         })
     }
 
+    fn prepare_preprocess<M: IntoIterator<Item = Self::PrepareMessage>>(
```

Comment:
I think we should be consistent about using either `HitsPrepareMessage` or `Self::PrepareMessage`. I lean toward `HitsPrepareMessage` because it makes it easier to jump to the concrete type definition.

Reply:
Done